cli.js

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env node
+const inquirer = require('inquirer');
+inquirer.registerPrompt('fuzzypath', require('inquirer-fuzzy-path'));
+
+const fs = require('fs');
+const path = require('path');
+const htmlToPdfCI = require('./index');
+const package = require('./package.json');
+const ci = process.argv[2];
+let configFile = {};
+try {
+    configFile = JSON.parse(fs.readFileSync(path.join(process.cwd(), '.htmltopdf.json')));

```

Comment:
What does `configFile` equal when fs fails and then it is parsed? Would it make sense to manually set configFile back to `{}` in the catch?

Reply:
Maybe also log that there's no config file?

Reply:
If `fs` fails to find the file, it throws an error. So in that case, the missing file would never be parsed.